internal/cmd/config/set.go

```diff
@@ -0,0 +1,121 @@
+package config
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/hetznercloud/cli/internal/cmd/cmpl"
+	"github.com/hetznercloud/cli/internal/cmd/util"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/cli/internal/state/config"
+)
+
+func NewSetCommand(s state.State) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "set <key> <value>...",
+		Short:                 "Set a configuration value",
+		Long:                  "Set a configuration value. For a list of all available configuration options, run 'hcloud help config'.",
```

Comment:
I think we should document the accepted values for durations. This is not obvious for non-go devs.

Reply:
Adding a quick description in either `hcloud config --help` or `hcloud config set --help` sounds good. While this is very intuitive, you need to know that it exists in the first place. People could expect that this is seconds, milliseconds, ...